lepton/starterkits/CONTRIBUTING.md

```diff
@@ -34,6 +34,23 @@ Starter Kits aim to spark developer interest and get the creative juices flowing
 
 This section lists all of the requirements for individual notebooks.
 
+### Formatting
+
+Notebooks should have a H1-level heading for the titles in the first cell (ie. Title starts with `# This Is My Title`). Logical sections within the notebook should be divided with H3-level headings (ie. `### Training Configuration`). If additional subdivisions are required, add H4 or lower level headings. All headings should be in Title Case.
```

Comment:
Maybe another example that mixes in the AP style. 

`A Title About the Correct Use of Titles`

Reply:
Updated, thanks for the suggestion!